content/construction/space_park_way_closure.md

```diff
@@ -11,22 +11,20 @@ draft = false
 
 +++
 
-In the summer of 2024, Sobrato will be grinding down Space Park Way and putting down new asphalt after renovating underground infrastructure for the new apartments. Completion of the project is scheduled for August 31, 2024.
+## Space Park Way Renovation
 
-The renovation is expected to take 2 or 3 weeks. During that time, the construction team intends to keep two-thirds of the street open so that Santiago Villa residents can continue to enter and exit the park.
+This summer, Sobrato will be grinding down Space Park Way and putting down new asphalt after renovating the infrastructure underground. The project is scheduled for completion by August 31, 2024. 
 
-### What you need to know
+### Parking will not be permitted on the north side of the street from July 15 through August 29.
```

Comment:
This doesn't feel like a headline to me.

Reply:
Yeah, I know. I just wanted to make it jump right out at the readers, because it's sort of the title of the whole thing. The other choice I had in mind was to just bold it underneath "What you need to know", which would make it a very strange bold paragraph under a bold heading. Suggest an alternative, maybe?

Reply:
I looked at this again on the VScode preview. It doesn't read as a heading, but it does stand out as the most important sentence in the article. 

So I'd like to keep it as is. I'll resolve this conversation, but I'm open to changing it later.